ntc_templates/templates/alcatel_sros_show_lag_port.textfsm

```diff
@@ -0,0 +1,21 @@
+Value LAG_ID (\d+\(\w\))
+Value PORT_ID (\d+/\d+/\d+)
+Value ADM (up|down)
+Value ACT_STD (active|standby)
+Value OPR (up|down)
+Value PRIMARY (yes|no)
+Value SUB_GROUP (\d+)
+Value FORCED (-)
+Value PRIO (\d+)
+
+Start
+  ^----------- -> LagPort
+
+LagPort
+  ^${LAG_ID}\s+${PORT_ID}\s+${ADM}\s+${ACT_STD}\s+${OPR}\s+${PRIMARY}\s+${SUB_GROUP}\s+${FORCED}\s+${PRIO}(\s|$$) -> Record
+  ^\s+${PORT_ID}\s+${ADM}\s+${ACT_STD}\s+${OPR}\s+${SUB_GROUP}\s+${FORCED}\s+${PRIO}(\s|$$) -> Record
+  ^\s*$$
+  ^-----------
+  ^===========
```

Comment:
```suggestion
  ^-+
  ^=+
```

Reduce characters in the horizontal separator lines by using regex with a plus sign

Reply:
Fixed!